client/js/_author/components/bank_navigation/bank_list.jsx

```diff
@@ -47,7 +47,7 @@ export default function bankList(props) {
           sortPublished={props.sortPublished}
         />
         <div className="au-c-scrollable">
-          { _.isEmpty(props.banks) ? <Spinner /> : items }
+          { props.banksLoaded ? items : <Spinner /> }
```

Comment:
I don't think I have an answer for that. The bug report doesn't say it should do that. I will ask though.